CIP-0027.md

```diff
@@ -0,0 +1,76 @@
+## Simple Summary
+
+Create a standard tagging for royalties&#39; distribution within the metadata of Cardano Assets.
+
+## Abstract
+
+This proposed tagging standard will allow for uniform royalties&#39; distributions. It has been developed with input from Artano, CNFT.io, Digital Syndicate, and Buffy Bot.
+
+## Motivation
+
+There is a significant number of artists who have expressed desire to leverage a royalties function within Cardano Assets, which will allow them a percentage of the future resale of assets they have created. There is also a misperception that Smart Contracts will have a built in solution for royalties. As of the writing of this CIPS proposal, Smart Contracts do not have this capability. By creating a standard for metadata tags, it will allow for immediate adoption, and backwards compatibility.
+
+## Specification
+
+&quot;royalties&quot; - Placed within CNFT Metadata.
+ Can either be an array that contains the royalties instructions or contains a link to offchain royalties json which includes the royalties instructions. The link to an offchain json is recommended so that it can be modified in the future.
+
+Royalties instructions tags:
+
+&quot;royaltiesContact&quot;: contact info for royalties inquiries (optional)
+ &quot;royaltiesInstructionVersion&quot;: version of formatting used
+ &quot;royaltiesPercent&quot;: Overall percentage of resale to be distributed as royalties
+ &quot;royaltiesPrimaryOwner&quot;: Friendly name of first royalties party (optional)
+ &quot;royaltiesPrimaryPercent&quot;: Percentage of royalties distribution for the first party
+ &quot;royaltiesPrimaryWallet&quot;: Wallet address to forward royalties to first party
+ &quot;royaltiesSecondaryOwner&quot;: Friendly name of second royalties party (optional)
+ &quot;royaltiesSecondaryPercent&quot;: Percentage of royalties distribution for the second party (optional)
+ &quot;royaltiesSecondaryWallet&quot;: Wallet address to forward royalties to second party (optional)
+ &quot;royaltiesTertiaryOwner&quot;: Friendly name of third royalties party (optional)
+ &quot;royaltiesTertiaryPercent&quot;: Percentage of royalties distribution for the third party (optional)
+ &quot;royaltiesTertiaryWallet&quot;: Wallet address to forward royalties to third party (optional)
+ &quot;royaltiesQuaternaryOwner&quot;: Friendly name of fourth royalties party (optional)
+ &quot;royaltiesQuaternaryPercent&quot;: Percentage of royalties distribution for the fourth party (optional)
+ &quot;royaltiesQuaternaryWallet&quot;: Wallet address to forward royalties to fourth party (optional)
+ 
+## Example JSON
+
+Onchain
```

Comment:
Isn't becoming centralised then? What happens if the URL and endpoint disappear or government blocks it?

Reply:
This might be one of the rare cases where being centralized isnt necessarily a bad thing.  As far as being blocked, the person implementing the on chain tag should use a shortened url that can be adjusted later on.  Bitly is a good example of that.

I can see use cases where the people in charge of the royalties handling is a trust.  I can also see there being some non profits that currate these royalties management.  The reason being, these royalties should live on in perpetuity.  We have to find a method that can evolve over time.  The only way to make that possible is for the record to be non immutable.

I'll mark this as resolved.  But if you have more question, comments, or concerns, please let me know.